src/NuGet.Clients/PackageManagement.UI/Xamls/PackageManagerControl.xaml.cs

```diff
@@ -630,7 +680,10 @@ private void Filter_SelectionChanged(object sender, EventArgs e)
             }
         }
 
-        internal void UpdatePackageStatus()
+        /// <summary>
+        /// Refreshes the control after packages are installed or uninstalled.
+        /// </summary>
+        private void Refresh()
```

Comment:
Renamed since the method does more than updating package status. It also refreshes the detail.